sailor/sap_iot/write.py

```diff
@@ -68,22 +70,82 @@ def _upload_data_single_indicator_group(dataset, indicator_set, group_id, templa
         _upload_data_single_equipment(data_subset, equipment.id, tags)
 
 
-def upload_indicator_data(dataset: TimeseriesDataset):
+def _check_indicator_group_is_complete(uploaded_indicators, indicator_group_id, template_id):
+    missing_indicators = []
+    request_url = _ac_application_url() + VIEW_TEMPLATES + '/' + template_id
+    oauth_ac = get_oauth_client('asset_central')
+    template = oauth_ac.request('GET', request_url)
+
+    if len(template) == 0:
+        raise RuntimeError(f' No template found for template id {template_id}.')
+
+    if len(template) > 1:
+        LOG.warning(f'More than one template found for template id {template_id}. Selecting first.')
+
+    item = template[0]
```

Comment:
I guess it would be clearer to call the `template` in line 77 `templates`, as that is a list, and then call `item` here `template`? At least l 86 below is much easier to understand with `template` than with `item`.